pkg/generate/apigwv2_test.go

```diff
@@ -184,77 +183,4 @@ func TestAPIGatewayV2_Route(t *testing.T) {
 		"RouteID",
 	}
 	assert.Equal(expStatusFieldCamel, attrCamelNames(statusFields))
-
-	expCreateOutput := `
-	if resp.ApiGatewayManaged != nil {
-		ko.Status.APIGatewayManaged = resp.ApiGatewayManaged
-	}
-	if resp.RouteId != nil {
-		ko.Status.RouteID = resp.RouteId
-	}
-`
-	assert.Equal(expCreateOutput, crd.GoCodeSetOutput(model.OpTypeCreate, "resp", "ko", 1, false))
-
-	expReadOneOutput := `
-	if resp.ApiGatewayManaged != nil {
```

Comment:
I see lots of tests removed. Was that intentional? Sometimes I remove them until the code gets ready and forget to add them back, so just wanted to check :)

Reply:
@muvaf nope, I just moved the tests to different files. The previous `GoCodeSetInput` checks were moved to `pkg/generate/code/set_sdk_test.go`. `GoCodeSetOutput` were moved to `pkg/generate/code/set_resource_test.go`, etc... :)